rules/defining-pbis/rule.md

```diff
@@ -11,45 +11,43 @@ related: []
 created: 2022-10-18T03:27:00.098Z
 guid: da1579f4-b598-4a2c-8fed-0b9b2230f6e8
 ---
-It is very common that a developer jumps onto a team and is given a PBI to work on, only to find out that it has almost no information. When that happens, it is crucial for the developer to raise their voice and gather enough information to be able to work on it.
+It is very common that a developer looks at a PBI to work on, only to find out that it has almost no information. When that happens, it is crucial for the developer to raise their voice and gather enough information so it meets the Definition of Ready.
```

Comment:
can you add a link to definition of ready and  remove link further down

Reply:
done